doc/integration/InputFormat.md

```diff
@@ -0,0 +1,31 @@
+# Lumen data input format
+
+Lumen currently supports import of data in CSV format either from a local file or from a web address (URL). Lumen will import your data and, to it's best capacity, guess the correct datatype of the CSV columns when importing it, thus saving you time in the data cleaning step.
+
+## File format
+
+A UTF-8 encoded file or file stream containing values separated by a comma `,`.
+
+The first row can include column headers or not. If the file is imported without column headers, these will get names generated automatically (`Column 1, Column 2, Column 3, etc`). (The column names can be changed by the user in the user interface.)
+
+## File source
+A file on your local hard drive or a file stream on a URL with http or https.
+
+## How to import data into Lumen
+CSV data is imported using `New->Dataset` in the Lumen user interface. Functionality will be added to refresh external data, both manually and on a schedule.

```

Comment:
@nadiagorchakova are we not planning to publish a lumen manual where the user interface (and features) is described?